src/service/userDetails.js

```diff
@@ -460,21 +470,23 @@ class UserDetails {
     });
   }
 
-  static _createTokenPair(userId, persona) {
-    // if user is found and password is valid
-    // create a token
+  static _createTokenPair(userId, persona, platform) {
+    let isMobilePlatform = platform && (_.includes(platform.toLowerCase(), 'ios') || _.includes(platform.toLowerCase(), 'android'))
+
     const userInfo = {
       id: userId,
       persona,
     };
     // TODO: [STYLE] Move the salt to a common location so that it can be reused
+    // expiresIn format: seconds (days * hours * minutes * seconds)
     const token = jwt.sign(userInfo, 'secret cant tell', {
-      expiresIn: 86400, // expires in 24 hours
+      expiresIn: isMobilePlatform ? (180 * 24 * 60 * 60) : (1 * 24 * 60 * 60),
     });
 
     // TODO: [STYLE] Move the salt to a common location so that it can be reused
+    // expiresIn format: seconds (days * hours * minutes * seconds)
     const refreshToken = jwt.sign(userInfo, 'secret cant tell', {
-      expiresIn: 30 * 24 * 60 * 60, // expires in 30 days
+      expiresIn: isMobilePlatform ? ((180 + 30) * 24 * 60 * 60) : (30 * 24 * 60 * 60),
```

Comment:
suggestion: have a local variable whose value is decided based on the `isMobilePlatform`. Then use it in the multiplier - easier to understand

Reply:
Also - move all the magic numbers into a constants file and reuse in all the 3 locations.